supermarq-benchmarks/examples/qre-challenge/Lessons-Learned.md

```diff
@@ -0,0 +1,100 @@
+# Architecture-Level Quantum Resource Estimator
+
+## Spencer Dearman
+
+### **Abstract**
+
+This document outlines the design and development of an architecture-level Quantum Resource Estimation (QRE) tool. Unlike existing QRE tools, which are mainly focused on quantum algorithm-level estimations, this tool aims to address the complexities involved in quantum architecture.
+
+
+### **Background and Motivation**
+
+Existing QRE tools are limited in their ability to provide insights at the architecture level, particularly when comparing Surface Codes with LRESC. Current tools just highlight the inherent advantage of LRESC in saving qubits, a conclusion that can be reached without extensive QRE analysis. This project aims to build a more advanced QRE tool that considers other factors such as Movement, Atom Reloading, Code Teleportation, Connectivity, Degree of Parallelization, and other architectural assumptions, providing more meaningful and actionable insights.
+
+ 
+### **Current Resource Estimation Limitations**
+
+_Limitations of Azure Quantum Resource Estimator and Qualtran_
+
+Both the Azure Quantum Resource Estimator (AQRE) and Qualtran are prominent tools in quantum resource estimation. Despite their widespread use, these tools are primarily algorithm-centric, with limited flexibility. By default, they assume a surface quantum error correction code with lattice surgery and magic state distillation, offering no customization of these parameters. In our research, we identified alternative QEC configurations that diverge from MSD and lattice surgery; however, testing these configurations was not feasible due to the tools’ rigid, hard-coded options.
```

Comment:
We want to confirm and claim that all resource estimators (BenchQ and pyLIQTR)  suffer from this problem